packages/cli/src/util/getHttpOperations.ts

```diff
@@ -0,0 +1,43 @@
+import { transformOas2Operation, transformOas3Operation } from '@stoplight/http-spec';
```

Comment:
Logic in this file should be extracted to the same repo all that ref resolving stuff will go. Doesn't make sense to keep it here imo.

Reply:
Curious, are you removing that logic from Graphite as well?

Reply:
If you don't want to move this code to individual repo then may consider moving to a different package? The JS API users will want to use this function too. Keeping this in `CLI` probably isn't the best place.

Reply:
1. I'm not removing this from Graphite — I think this glue code is still used there somewhere. Also, this is not an exact copy and paste of what's in Graphite.
2. Probably you're right, this is not the right place. I *could* bring it in `core` but I'd like to avoid it if possible; in general I want people to reason about Prism with arrays of `HttpOperation`, and not with files, web resources and stuff like that.

Reply:
Sure, it's not a big deal!